pages/serializers.py

```diff
@@ -255,6 +255,17 @@ def get_path(self, obj):
             slug = obj.get('slug', "")
         return prefix + slug
 
+    @staticmethod
+    def get_text(obj):
```

Comment:
The `content_format` to HTML conversion is done server-side. What happens if we want to edit the `PageElement.text` as Markdown?
Since the Update Serializer is passed a `content_format` and `text` field (formatted based on the content_format), it could make sense to do the same on the Retrieve API. The code here could be used to add an `html_formatted` read-only field for front-end client convienience.